doc/nrf-bm/libraries/nfc/index.rst

```diff
@@ -0,0 +1,78 @@
+.. _lib_nfc:
+
+NFC libraries
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The NFC libraries support NFC-A listen mode, where the tag waits for polling devices but does not actively start a connection.
+This makes them ideal for implementing NFC tag functionality in bare metal applications.
+
+.. note::
+   For comprehensive information about NFC and NDEF libraries in the nRF Connect SDK, see the `nRF Connect SDK NFC libraries`_.
```

Comment:
Should it somehow be clarified that not all that is described here can be used in NCS BM context?